polaris-react/tests/setup/tests.ts

```diff
@@ -6,3 +6,8 @@ import './matchers';
 afterEach(() => {
   destroyAll();
 });
+
+globalThis.requestAnimationFrame = (cb) => {
```

Comment:
I think this breaks some other tests that implicitly depend on this being a pure stub rather than mocking the implementation. But this is needed for the Scrollable component to actually be tested properly. I can move this to the test file and keep from disrupting other tests, but it seems something like this (or a better shim) should be in all test runs so components that use requestAnimationFrame actually exercise the code in tests.

What to do?